src/fidesops/ops/service/masking/strategy/masking_strategy_string_rewrite.py

```diff
@@ -10,23 +9,26 @@
 )
 from fidesops.ops.service.masking.strategy.format_preservation import FormatPreservation
 from fidesops.ops.service.masking.strategy.masking_strategy import MaskingStrategy
-from fidesops.ops.service.masking.strategy.masking_strategy_factory import (
-    MaskingStrategyFactory,
-)
+from fidesops.ops.service.strategy_factory import register
 
+# this constant is kept around because it is referenced in generated alembic migrations
 STRING_REWRITE_STRATEGY_NAME = "string_rewrite"
```

Comment:
Instead of importing this into that migration then, I might just move this string directly into the migration.